### DISCUSSION TITLE
Support for BC7 format compressed textures

### DISCUSSION THREAD

**Discussion by EsinVitaliy** [MonoGame/MonoGame#8141] [Graphics]
Are there plans to add the BC7 compression format import for Windows apps?
I can create BC7 compressed alpha texture with **Compressonator**, but I cannot import it with **MGCB Editor**.
Error on build:
```
Importer 'TextureImporter' had unexpected failure!
Microsoft.Xna.Framework.Content.ContentLoadException: Unsupported DDS_HEADER_DXT10 struct found
```

**Comment by ThomasFOG** [comment]
We have plans to replace the current texture compressor with Compressonator on Windows and Linux (and TextureConverter on macOS). See #8124.

However, we still don't know if we'll expose each and every supported format. We are likely to do a first pass with only the current supported formats, but past that point it'll be easier to add more or accept community contributions of new formats.

Supporting texture formats is not just about compressing a file into another one, it also needs to make the code and graphical backend aware of which compression is going to be uploaded to the GPU.
